python/ray/air/_internal/torch_utils.py

```diff
@@ -159,7 +171,7 @@ def convert_ndarray_batch_to_torch_tensor_batch(
 
     Returns: A (dict of) Torch Tensor(s).
     """
-    if isinstance(ndarrays, np.ndarray):
+    if not isinstance(ndarrays, dict):
```

Comment:
Is `ndarrays` always `np.ndarray | dict[str, np.ndarray]` like the type hint suggests? If so, isn't this equivalent to the old code?

Reply:
reverted back